### ISSUE TITLE
can I specify that the image resolution ratio is 720x1440?

### ISSUE THREAD

**Issue by github-water** [ostris/ai-toolkit#210]
## This is for bugs only

Did you already ask [in the discord](https://discord.gg/VXmU2f5WEU)?

Yes/No

You verified that this is a bug and not a feature request or question by asking [in the discord](https://discord.gg/VXmU2f5WEU)?

Yes/No

**Comment by vancyland** [comment]
yes, no problem. Go inside ./ai-toolkit-main/toolkit/buckets.py file, edit the resolutions_1024 list and you can simply customize your resolution.
